cmake/modules/platform/toolcfg/gnu.cmake

```diff
@@ -110,13 +127,25 @@ function(_omr_toolchain_separate_debug_symbols tgt)
 	else()
 		omr_get_target_output_genex(${tgt} output_name)
 		set(dbg_file "${output_name}${OMR_DEBUG_INFO_OUTPUT_EXTENSION}")
-		add_custom_command(
-			TARGET "${tgt}"
-			POST_BUILD
-			COMMAND "${CMAKE_OBJCOPY}" --only-keep-debug "${exe_file}" "${dbg_file}"
-			COMMAND "${CMAKE_OBJCOPY}" --strip-debug "${exe_file}"
-			COMMAND "${CMAKE_OBJCOPY}" --add-gnu-debuglink="${dbg_file}" "${exe_file}"
-		)
+		if(OMR_OS_AIX AND CMAKE_C_COMPILER_IS_OPENXL)
+			add_custom_command(
+				TARGET "${tgt}"
+				POST_BUILD
+				#COMMAND "${CMAKE_OBJCOPY}" --only-keep-debug "${exe_file}" "${dbg_file}"
+				#COMMAND "${CMAKE_OBJCOPY}" --strip-debug "${exe_file}"
+				#COMMAND "${CMAKE_OBJCOPY}" --add-gnu-debuglink="${dbg_file}" "${exe_file}"
```

Comment:
Is there a reason to keep these?

Reply:
Nope, removed.

Reply:
Please keep them, (I thought I had commented that) they serve as a reminder of work to be done to separate debug information.

Reply:
Perhaps that's better conveyed in a PR rather than as a comment in code/config.

Reply:
That works for me.